### ISSUE TITLE
Laggy launcher if tracking is on

### ISSUE THREAD

**Issue by Hoerli1337** [mendhak/gpslogger#1183]
Hello!

I use GPSLogger as a replacement for Fine my device from Google.
I let the app start and stop automatically via Tasker.
The data is loaded into a NextCloud.

Unfortunately, I've noticed that my launcher (Google Pixel 8 Pro - Android 15) is sometimes extremely sluggish and sometimes doesn't respond at all when GPSLogger is running.
If I close the app via the status bar, everything works fine again. 

I noticed this behavior with Android 13, 14 and now with Android 15.
Is this normal?
Can anything be done about it?
The app itself works normally and sends the location data to NextCloud.
It's just extremely annoying when you need the phone and it takes 30 seconds to remember before you can use it.

**Comment by Hoerli1337** [comment]
I have set the recording interval to 60 seconds.
The GPS does not need to be on in between.
Distance between measuring points: 3 meters
Accuracy: 40 meters
Time until accuracy is reached: 60 seconds

I have actually set it quite relaxed so as not to put an extreme strain on the battery.

I can try to create a debug log, but I can't promise that it will contain anything useful.
... may take some time, as the error is not immediately reproducible.

**Comment by mendhak** [comment]
Yeah those settings look relaxed, so I can't see it being that. Could it be the app itself causing it, I wonder. 

So, if you launch GPSLogger via Tasker, versus directly, is there a difference?

Is it only sluggish when syncing to NextCloud? (Maybe the log view will give a clue) What happens if you turn off NextCloud syncing.
